notebooks/pulseplate_rag_release_gates.ipynb

```
@@ -0,0 +1,1528 @@
+    "GENERATOR_MODE = os.getenv(\"GENERATOR_MODE\", \"extractive_stub\").strip().lower()\n",
+    "\n",
+    "ENABLE_NLI_MODEL = os.getenv(\"ENABLE_NLI_MODEL\", \"false\").strip().lower() in {\n",
+    "    \"1\",\n",
+    "    \"true\",\n",
+    "    \"yes\",\n",
+    "    \"on\",\n",
+    "}\n",
+    "NLI_MODEL_NAME = os.getenv(\"NLI_MODEL_NAME\", \"roberta-large-mnli\")\n",
+    "\n",
+    "# Release-gate thresholds from your requested plan.\n",
+    "GATE_THRESHOLDS = {\n",
```

Flake8 (7.3.0)

[error] undefined name 'display'

(F821)